### PR TITLE
Update CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by ghost**
Add WordPress.org Username.
Follow up to #1350

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
-| @brickbones | |
-| @iamgabrielma | |
-| @swissspidy | |
-| @dixitadusara | |
-| @ameeker | |
-| @StaggerLeee | |
-| @jblz | |
+| @spocke | @spocke |
+| @androb | @androb |
+| @annaephox | :mag: :tired_face: |
```
Comment (desrosj) [inline CONTRIBUTORS.md]: There are several of these. Can they be removed, or the correct usernames be added?

**Review by desrosj — CHANGES_REQUESTED** [review]
I spot checked a few. Thanks for working on this, @pratikthink!

File: CONTRIBUTORS.md (hunk 1 above)
Comment (pratikthink) [reply CONTRIBUTORS.md]: I am not able to find those (:mag: :tired_face:) usernames on WordPress.org with the help of full name, avatar, web URLs and also Google.

Comment (pratikthink) [reply CONTRIBUTORS.md]: I don't know how it can be solved.

Comment (Soean) [inline CONTRIBUTORS.md]:
```suggestion
| @annaephox | @annaharrison |
```

Comment (Soean) [reply CONTRIBUTORS.md]: Nice work! If you can't find a wp.org account leave it empty. Maybe some of them don't have one.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
+| @buzztone | @buzztone |
+| @mathetos | @webdevmattcrom |
+| @GaryJones | @garyj |
+| @jasonagnew | :mag: :tired_face: |
+| @brickbones | @ieatwebsites |
+| @iamgabrielma | @gma992 |
+| @swissspidy | @swissspidy |
+| @dixitadusara | :mag: :tired_face: |
+| @ameeker | @ameeker |
+| @StaggerLeee | :mag: :tired_face: |
```
Comment (Soean) [inline CONTRIBUTORS.md]:
```suggestion
| @StaggerLeee | @stagger-lee |
```

File: CONTRIBUTORS.md (hunk 1 above)
Comment (pratikthink) [reply CONTRIBUTORS.md]: ok

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
-| @paulwilde | |
-| @mitogh | |
-| @codebykat | |
+| @youknowriad | @youknowriad |
+| @aduth | @aduth |
+| @jasmussen | @joen |
+| @iseulde | @iseulde |
+| @mtias | @mtias |
+| @nylen | @jnylen0 |
+| @EphoxJames |  |
```
Comment (desrosj) [inline CONTRIBUTORS.md]:
```suggestion
| @EphoxJames | |
```

File: CONTRIBUTORS.md [hunk 4]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
+| @mkaz | @mkaz |
+| @notnownikki | @notnownikki |
+| @BE-Webdesign | @chopinbach |
+| @njpanderson |  |
```
Comment (desrosj) [inline CONTRIBUTORS.md]:
```suggestion
| @njpanderson | |
```

File: CONTRIBUTORS.md [hunk 5]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
+| @mimo84 |  |
```
Comment (desrosj) [inline CONTRIBUTORS.md]:
```suggestion
| @mimo84 | |
```

File: CONTRIBUTORS.md [hunk 6]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
+| @intronic |  |
```
Comment (desrosj) [inline CONTRIBUTORS.md]:
```suggestion
| @intronic | |
```

File: CONTRIBUTORS.md [hunk 7]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
-| @Soean | |
-| @mapk | |
-| @sirjonathan | |
-| @j-falk | |
-| @ryelle | |
+| @kopepasah | @kopepasah |
+| @circlecube | @circlecube |
+| @adamsilverstein | @adamsilverstein |
+| @timmyc | @timmydcrawford |
+| @ephox-mogran |  |
```
Comment (desrosj) [inline CONTRIBUTORS.md]:
```suggestion
| @ephox-mogran | |
```

File: CONTRIBUTORS.md [hunk 8]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
+| @lancewillett | @lancewillett |
+| @lynneux | @lynneux |
+| @betsela | @betsela |
+| @fuyuko | @fuyuko |
+| | @msdesign21 |
+| @thrijith | @thrijith |
+| @Cloud887 |  |
+| @hblackett | @hblackett |
+| @vishalkakadiya | @vishalkakadiya |
+| @c-shultz |  |
```
Comment (desrosj) [inline CONTRIBUTORS.md]:
```suggestion
| @c-shultz | |
```

File: CONTRIBUTORS.md [hunk 9]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
-| @vishalkakadiya |
-| @c-shultz |
-| @nfmohit-wpmudev |
+| @lancewillett | @lancewillett |
+| @lynneux | @lynneux |
+| @betsela | @betsela |
+| @fuyuko | @fuyuko |
+| | @msdesign21 |
+| @thrijith | @thrijith |
+| @Cloud887 |  |
```
Comment (desrosj) [inline CONTRIBUTORS.md]:
```suggestion
| @Cloud887 | |
```

File: CONTRIBUTORS.md [hunk 10]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
+| @iamgabrielma | @gma992 |
+| @swissspidy | @swissspidy |
+| @dixitadusara |  |
+| @ameeker | @ameeker |
+| @StaggerLeee | @stagger-lee |
+| @jblz | @jblz |
 | @nic-bertino | @nicbertino |
 | @rahmon | @rahmohn |
-| @vladanost | |
+| @vladanost |  |
```
Comment (desrosj) [inline CONTRIBUTORS.md]:
```suggestion
| @vladanost | |
```

File: CONTRIBUTORS.md [hunk 11]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
+| @hugobaeta | @hugobaeta |
+| @mizejewski | @mizejewski |
+| @buzztone | @buzztone |
+| @mathetos | @webdevmattcrom |
+| @GaryJones | @garyj |
+| @jasonagnew |  |
+| @brickbones | @ieatwebsites |
+| @iamgabrielma | @gma992 |
+| @swissspidy | @swissspidy |
+| @dixitadusara |  |
```
Comment (desrosj) [inline CONTRIBUTORS.md]:
```suggestion
| @dixitadusara | |
```

File: CONTRIBUTORS.md [hunk 12]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
+| @mrwweb | @mrwweb |
+| @diegoliv | @diegoliv |
+| @lukecav | @lukecavanagh |
+| @shaunandrews | @shaunandrews |
+| @hugobaeta | @hugobaeta |
+| @mizejewski | @mizejewski |
+| @buzztone | @buzztone |
+| @mathetos | @webdevmattcrom |
+| @GaryJones | @garyj |
+| @jasonagnew |  |
```
Comment (desrosj) [inline CONTRIBUTORS.md]:
```suggestion
| @jasonagnew | |
```

File: CONTRIBUTORS.md [hunk 13]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
+| @annaephox | @annaharrison |
+| @Afraithe |  |
+| @georgeh |  |
+| @m | @matt |
+| @melchoyce | @melchoyce |
+| @pento | @pento |
+| @karmatosed | @karmatosed |
+| @nitrajka | @nitrajka |
+| @sirreal |  |
+| @inhil |  |
```
Comment (desrosj) [inline CONTRIBUTORS.md]:
```suggestion
| @inhil | |
```

File: CONTRIBUTORS.md [hunk 14]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
+| @androb | @androb |
+| @annaephox | @annaharrison |
+| @Afraithe |  |
+| @georgeh |  |
+| @m | @matt |
+| @melchoyce | @melchoyce |
+| @pento | @pento |
+| @karmatosed | @karmatosed |
+| @nitrajka | @nitrajka |
+| @sirreal |  |
```
Comment (desrosj) [inline CONTRIBUTORS.md]:
```suggestion
| @sirreal | |
```

File: CONTRIBUTORS.md [hunk 15]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
-| @swissspidy | |
-| @dixitadusara | |
-| @ameeker | |
-| @StaggerLeee | |
-| @jblz | |
+| @spocke | @spocke |
+| @androb | @androb |
+| @annaephox | @annaharrison |
+| @Afraithe |  |
+| @georgeh |  |
```
Comment (desrosj) [inline CONTRIBUTORS.md]:
```suggestion
| @georgeh | |
```

File: CONTRIBUTORS.md [hunk 16]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
-| @iamgabrielma | |
-| @swissspidy | |
-| @dixitadusara | |
-| @ameeker | |
-| @StaggerLeee | |
-| @jblz | |
+| @spocke | @spocke |
+| @androb | @androb |
+| @annaephox | @annaharrison |
+| @Afraithe |  |
```
Comment (desrosj) [inline CONTRIBUTORS.md]:
```suggestion
| @Afraithe | |
```

**Review by desrosj — COMMENTED** [review]
Few more small changes. Let's not add spaces to the lines I marked. We are no longer changing those lines, so let's leave them the same.

File: CONTRIBUTORS.md [hunk 17]
```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
...
-| @mrwweb | |
-| @diegoliv | |
-| @lukecav | |
-| @shaunandrews | |
-| @hugobaeta | |
-| @mizejewski | |
-| @buzztone | |
-| @mathetos | |
-| @GaryJones | |
+| @georgeolaru | @@babbardel |
```
Comment (ZebulanStanphill) [inline CONTRIBUTORS.md]: Looks like there is a stray `@` here.
```suggestion
| @georgeolaru | @babbardel |
```

**Review by ZebulanStanphill — CHANGES_REQUESTED** [review]

**Review by Soean — APPROVED** [review]
LGTM

**Review by ZebulanStanphill — APPROVED** [review]

**Comment by pratikthink** [comment]
I am the first-timer and it was a nice experience for me.

File: CONTRIBUTORS.md [hunk 18]
```diff
@@ -106,10 +106,11 @@ This list is manually curated to include valuable contributions by volunteers th
...
+| @danielbachhuber | @danielbachhuber |
+| @jorgefilipecosta | @jorgefilipecosta |
+| @ajitbohra | @ajitbohra |
+| @ChrisVanPatten | @chrisvanpatten |
 | @mayukojpn | @mayukojpn |
 | @tofumatt | @lonelyvegan |
-| @LukePettway | @luke_pettway |
\ No newline at end of file
+| @LukePettway | @luke_pettway |
+| @pratikthink| @pratikthink |
```
Comment (ZebulanStanphill) [inline CONTRIBUTORS.md]: Minor nitpick, but there seems to be a missing space here.
```suggestion
| @pratikthink | @pratikthink |
```

**Review by desrosj — APPROVED** [review]

**Comment by desrosj** [comment]
Thank you, @pratikthink! Hope to see you around some more.

**Pull request merged**